File: 백준/Bronze/5597. 과제 안 내신 분．．？/과제 안 내신 분．．？.cs
List<int> numbers = new List<int> {};

for(int i = 1; i <= 30; i++)
{
    numbers.Add(i);
}

for (int i = 1; i <= 28; i++)
{
    int num = int.Parse(Console.ReadLine());
    if(numbers.Contains(num))
    {
        numbers.Remove(num);
    }
}

Console.WriteLine(numbers.Min());
Console.WriteLine(numbers.Max());

File: 백준/Bronze/5597. 과제 안 내신 분．．？/과제 안 내신 분．．？.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        List<Integer> numbers = new ArrayList<>();
        for (int i = 1; i <= 30; i++) {
            numbers.add(i);
        }

        for (int i = 0; i < 28; i++) {
            int num = sc.nextInt();
            numbers.remove(Integer.valueOf(num));
        }

        Collections.sort(numbers);
        System.out.println(numbers.get(0));
        System.out.println(numbers.get(1));

        sc.close();
    }
}
